examples/EtherNetIP/EtherNetIPSimulation.py

```diff
@@ -0,0 +1,91 @@
+# -*- coding: utf-8 -*-
```

Comment:
This is not a simulation. Let us name it as CipSocketGraphite.py to indicate it is have Cip as DeviceComms, Socket as DCCComms and Graphite as DCC.

Also add the writer code as "simulator", so that someone can test this package. In Readme, also add the complete steps of how to run the simulator and load this package.

Reply:
okay